xlive/Blam/Engine/objects/objects.cpp

```diff
@@ -988,14 +988,17 @@ int16 __cdecl object_get_markers_by_string_id(datum object_index, string_id mark
 	return internal_object_get_markers_by_string_id(object_index, marker, marker_object, count, false);
 }
 
-// Replace calls to internal_object_get_markers_by_string_id
-void internal_object_get_markers_by_string_id_replace_calls(void)
+// Replace calls to functions getting the marker by string id
+void object_get_markers_by_string_id_replace_calls(void)
 {
 	// PatchCall(Memory::GetAddress(0x132792, 0x121662), internal_object_get_markers_by_string_id);	(Disabled as this is not interpolated)	TODO: Add parameter to internal_object_get_markers_by_string_id instead of not patchcalling
 	// PatchCall(Memory::GetAddress(0x1327B1, 0x121681), internal_object_get_markers_by_string_id);	(Disabled as this is not interpolated)	TODO: Add parameter to internal_object_get_markers_by_string_id instead of not patchcalling
-	PatchCall(Memory::GetAddress(0x134C26, 0x123AF6), internal_object_get_markers_by_string_id);
 	// PatchCall(Memory::GetAddress(0x13823D, 0x12710D), internal_object_get_markers_by_string_id);	(Disabled as this is not interpolated)	TODO: Add parameter to internal_object_get_markers_by_string_id instead of not patchcalling
 	// PatchCall(Memory::GetAddress(0x138257, 0x1214CB), internal_object_get_markers_by_string_id);	(Disabled as this is not interpolated)	TODO: Add parameter to internal_object_get_markers_by_string_id instead of not patchcalling
+	PatchCall(Memory::GetAddress(0x134C26, 0x123AF6), internal_object_get_markers_by_string_id);	// object_animation_callback
```

Comment:
this line hooks something audio related rather than visual, probably not even called outside game tick context

Reply:
It's used for an animation callback function, from what I can tell it has to do with animation events and it creates a sound at that position